server/lorax_server/models/flash_bert.py

```diff
@@ -97,7 +97,9 @@ def __init__(
             dtype,
             process_group=self.process_group,
         )
-        prefix = None if (model_id == "WhereIsAI/UAE-Large-V1") else "bert"
+        prefix = "bert"
```

Comment:
Is there a way to generalize this? For example, if the model is not a token classification model, or maybe we check the weight names, etc.?